### PR TITLE
Add port-common bit and KAT-OPEN-COMMON-PORT FindingType

### LINKED ISSUE #543: [Bug] Common open ports are not reported as findings
**Describe the bug**
Common ports such as 443 do not end up in KATs findings. However, not every service with port 443 open was also intended to have that port reachable by KAT. In a report, as an end-user I want to know all open ports that KAT has found.

**To Reproduce**
1. Scan a website
2. Generate a report
3. There is no finding for a common open port

**Expected behavior**
Report all open ports seen by KAT.

**OpenKAT version**
c799f181cdd4569304925f625affcf9f58092607 (latest)

**Additional context**
https://github.com/minvws/nl-kat-coordination/blob/82d64a588f641ab3551a7c338393235dfd0e4774/octopoes/bits/port_classification/port_classification.py#L35

### PR DESCRIPTION
**Pull request by dekkers**
### Changes
_Please describe the essence of this PR in a few sentences. Mention any breaking changes or required configuration steps._

### Issue link
Fixes #543

### Proof
_Please add some proof of your working change here, unless this is not required (e.g. this PR is trivial)._

---
### Code Checklist
- [ ] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
- [ ] The code does not violate Model-View-Template and our other architectural principles.
- [ ] The code prioritizes readability over performance where appropriate.
- [ ] The code does not bypass authentication or security mechanisms.
- [ ] The code does not introduce any dependency on a library that has not been properly vetted.
- [ ] The code contains docstrings, comments, and documentation where needed.

---
## Checklist for QA:
- [ ] I have checked out this branch, and successfully ran a fresh `make kat`.
- [ ] I confirmed that there are no unintended functional regressions in this branch:
    - [ ] I have managed to pass the onboarding flow
    - [ ] Objects and Findings are created properly
    - [ ] Tasks are created and completed properly
- [ ] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* _bullet point + screenshot (if useful) per tested functionality_

### What doesn't work:
* _bullet point + screenshot (if useful) per tested functionality_

### Bug or feature?:
* _bullet point + screenshot (if useful) if it is unclear whether something is a bug or an intended feature._

### CODE REVIEW

File: rocky/OOI_database_seed.json [hunk 1]
```diff
@@ -509,6 +509,19 @@
       "consult_api": false
     }
   },
+  {
+    "model": "tools.ooiinformation",
+    "pk": "KATFindingType|KAT-OPEN-COMMON-PORT",
+    "fields": {
+      "last_updated": "2021-08-09T14:50:09.835Z",
```
Comment (praseodym) [inline rocky/OOI_database_seed.json:516]: Why do we have a timestamp here if it doesn't actually reflect when the record was updated?

**Review by Lisser — APPROVED** [review]

File: rocky/OOI_database_seed.json (hunk 1 above)
Comment (dekkers) [reply rocky/OOI_database_seed.json:516]: The last_updated field isn't used anywhere just like the consult_api field, I think we should just delete the fields. We should probably just delete those fields...

Comment (underdarknl) [reply rocky/OOI_database_seed.json:516]: isn't consult api used to check for updates when dealing with cve's etc? Those do need periodic updates.

**Review by underdarknl — APPROVED** [review]

File: rocky/OOI_database_seed.json (hunk 1 above)
Comment (dekkers) [reply rocky/OOI_database_seed.json:516]: But it isn't used. The value is False by default, the value is False for everything in rocky/OOI_database_seed.json and there is no code that ever sets it to True as far as I can see.

Comment (noamblitz) [reply rocky/OOI_database_seed.json:516]: Yes true :). The idea was that when cve data was updated, people could use this to consult the cve database again (and other ooi information sources for that matter)

**Pull request merged**